Sources/RemindersLibrary/CLI.swift

```diff
@@ -12,6 +12,18 @@ private struct ShowLists: ParsableCommand {
     }
 }
 
+private struct ExportAll: ParsableCommand {
+    static let configuration = CommandConfiguration(
+        abstract: "Export all reminders as JSON")
+
+    @Flag(help: "Pretty print the JSON data")
+    var prettyPrint = false
```

Comment:
lets just remove this and default it to true, shouldn't be a major issue of performance or anything

Reply:
Hrmm, fair. I'd personally be inclined to change it to `noPrettyPrint` and default it to `true` rather than remove it outright as I favour tools that give me flexibility. But if that's a dealbreaker, I can remove it entirely.